python/src/python_aiconfig.egg-info/PKG-INFO

```diff
@@ -1,6 +1,6 @@
 Metadata-Version: 2.1
```

Comment:
Haven't gotten a chance to understand the process of publishing, you aren't using the `.egg` to publish right?